7.figures/nbconverted/Figure3_model_evaluation.r

```diff
@@ -347,13 +357,15 @@ fig_3_gg
 bottom_plot <- (
     pr_curve_subset_gg | 
     f1_score_subset_gg
-) + plot_layout(widths = c(3, 0.55))
+) + plot_layout(widths = c(3, 0.7))
 
 sup_fig_3_gg <- (
     wrap_elements(confusion_matrix_subset_gg) /
     bottom_plot
-) + plot_annotation(tag_levels = "A") + plot_layout(heights = c(1, 0.6))
+) + plot_annotation(tag_levels = "A") + plot_layout(heights = c(1, 0.5))
 
 ggsave(output_sup_figure_subset, dpi = 500, height = 14, width = 14)
 
 sup_fig_3_gg
+
```

Comment:
Just curious, how does Rscript formatting work, and should there be two empty endlines at the bottom of the script?

Reply:
nope! I have removed an extra empty cell in the .ipynb file. thanks for noting this!